feat: UIG-2506 - vl-select - disable search

ksvlaanderen opened the pull request:
data-vl-select-disable-search attribuut toegevoegd
Storybook verbeterd
Cypress uitgebreid

Goldflow commented on libs/elements/src/lib/select/stories/vl-select.stories-doc.mdx:
```diff
@@ -0,0 +1,42 @@
+</Canvas>
+
+## Referenties
+
```
Eventueel `choices.js` ook hier vermelden met link naar de originele repo / site

moet niet per sé, ik heb dit doorgaans gedaan bij andere components

ksvlaanderen replied:
Done!

kspeltix commented on libs/elements/src/lib/select/vl-select.lib.js:
```diff
@@ -4212,9 +4213,15 @@
             placeholderValue = vl.i18n.t('select.placeholder_value'),
             disabled = false; // Select has search - default behavior
 
+        // @deprecated: This is not how boolean attributes should work
```
best op deze manier deprecaten
/** @deprecated: This is not how boolean attributes should work */

ksvlaanderen committed:
feat: UIG-2506 - vl-select - disable search

data-vl-select-disable-search attribuut toegevoegd
Storybook verbeterd
Cypress uitgebreid

kspeltix commented:
:tada: This PR is included in version 1.10.0 :tada:

The release is available on [GitHub release](https://github.com/milieuinfo/uigov-web-components/releases/tag/v1.10.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
